src/components/directory-content.js

```diff
@@ -9,8 +9,10 @@ import { useCallback, useEffect, useMemo, useRef, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { setActiveDirectory, setSelectionForCopy } from '../redux/actions';
 import { FormattedMessage, useIntl } from 'react-intl';
+import { DialogsId } from '../utils/UIconstants';
```

Comment:
You don't need this import. **constants** contains all elements . You can use it by calling  **constant.DialogsId**